promptops/promptops/skill_export.py

```
@@ -186,10 +187,33 @@ def generate_health_dashboard(docs_path: Path, health_report: List[Dict[str, Any
 
     dashboard_path.write_text("\n".join(lines), encoding='utf-8')
 
+def cleanup_orphaned_skills(docs_path: Path, active_skills: Set[Path]):
+    out_skills_dir = docs_path / "skills"
+    if not out_skills_dir.exists():
+        return
+
+    active_paths = {p.resolve() for p in active_skills}
+
+    # Find all SKILL.md files
+    for root, dirs, files in os.walk(out_skills_dir, topdown=False):
```

Ruff (0.15.15)

[warning] Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)